plugins/modules/ios_ospfv3.py

```diff
@@ -0,0 +1,2032 @@
+                    type: dict
+                    suboptions:
+                      first_delay:
+                        description:
+                          - Delay to generate first occurrence of LSA in milliseconds
+                          - Note, refer vendor documentation for respective valid values
+                        type: int
+                      min_delay:
+                        description:
+                          - Minimum delay between originating the same LSA in milliseconds
+                          - Note, refer vendor documentation for respective valid values
+                        type: int
+                      max_delay:
+                        description:
+                          - Maximum delay between originating the same LSA in milliseconds
+                          - Note, refer vendor documentation for respective valid values
+                        type: int
+                  spf:
+                    description: OSPF SPF throttle timers
+                      - Delay between receiving a change to SPF calculation in milliseconds
+                      - Note, refer vendor documentation for respective valid values
+                    type: dict
+                    suboptions:
+                      receive_delay:
+                        description:
+                          - Delay between receiving a change to SPF calculation in milliseconds
+                          - Note, refer vendor documentation for respective valid values
+                        type: int
+                      between_delay:
+                        description:
+                          - Delay between first and second SPF calculation in milliseconds
+                          - Note, refer vendor documentation for respective valid values
+                        type: int
+                      max_delay:
+                        description:
+                          - Maximum wait time in milliseconds for SPF calculations
+                          - Note, refer vendor documentation for respective valid values
+                        type: int
+  running_config:
+    description:
+    - This option is used only with state I(parsed).
+    - The value of this option should be the output received from the IOS device by
+      executing the command B(sh running-config | section ^router ospfv3).
+    - The state I(parsed) reads the configuration from C(running_config) option and
+      transforms it into Ansible structured data as per the resource module's argspec
+      and the value is then returned in the I(parsed) key within the result.
+    type: str
+    choices:
+    - merged
+    - replaced
+    - overridden
+    - deleted
+    - gathered
+    - parsed
+    - rendered
+    default: merged
+
+"""
+EXAMPLES = """
+
+# Using deleted
+
+# Before state:
+# -------------
+#
+# router-ios#sh running-config | section ^router ospf
+# router ospf 200 vrf blue
```

Comment:
looks like ios is a bit different with cli . ospf means ospfv2 in some of other platforms.